src/aleph/vm/controllers/qemu_confidential/instance.py

```diff
@@ -0,0 +1,117 @@
+import asyncio
+import logging
+import shutil
+from asyncio.subprocess import Process
+from typing import Callable, Optional
+
+from aleph_message.models import ItemHash
+from aleph_message.models.execution.environment import MachineResources
+
+from aleph.vm.conf import settings
+from aleph.vm.controllers.configuration import (
+    Configuration,
+    HypervisorType,
+    QemuConfidentialVMConfiguration,
+    save_controller_configuration,
+)
+from aleph.vm.controllers.qemu import AlephQemuInstance
+from aleph.vm.controllers.qemu.instance import (
+    AlephQemuResources,
+    ConfigurationType,
+    logger,
+)
+from aleph.vm.network.interfaces import TapInterface
+
+logger = logging.getLogger(__name__)
+
+
+class AlephQemuConfidentialResources(AlephQemuResources):
+    pass
```

Comment:
```suggestion
    # TODO: Implement download of the custom OVMF bootloader to use if specified, if not only use the default one.
    pass
```

Reply:
we can probably have the default one specified in the client, like we do for runtime, so we would always download one (or take it from the cache)